test/OrchardCore.Tests/Data/JsonDynamicTests.cs

```diff
@@ -810,7 +810,7 @@ public void SerializingJsonDynamicValueMustWriteValueOnly()
         Assert.Equal("{\"stringValue\":\"test\",\"numberValue\":123,\"booleanValue\":true}", jsonStr);
     }
 
-    public class TestPart : ContentPart
+    public sealed class TestPart : ContentPart
```

Comment:
sealed is only useful for perf when it has methods overrides, or to prevent users from inheriting it. Here there is no reason to make these changes. The same way there is no reason to remove it now it's there.